Firebase/Core/Public/FIRVersion.h

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Putting FIRVersion.h may require an API review.  @ryanwilson ?

Reply:
I moved it to Private. It probably would make sense to move it to Public/ at some point though.